iulianIachimov/src/main/java/datacollectiontask/Models/Order.java

```diff
@@ -0,0 +1,53 @@
+package datacollectiontask.Models;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    public long id;
+    public LocalDateTime date;
+    public User user;
+    public List<Item> items;
+
+    private static long idCounter;
+
+    public Order(User user, List<Item> items) {
+        this.id = ++idCounter;
+        this.date = LocalDateTime.now();
+        this.user = user;
+        this.items = items;
+    }
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("hh:mm, MMMM dd, YYYY");
```

Comment:
static final, and move it above all methods

Reply:
sorry, what exactly needs to be moved to static final?
DateTimeFormatter is not being called at all - is it OK if I just delete it from the code?

Reply:
FINAL should be the date because it is not changed after it's set. So, each order should have only one date which is not going to be changed. Regarding static, I think there is no need for static date, beacuse date it's a instance related property that depend on object Order

Reply:
if you keep formatter, it should be static

Reply:
I deleted the formatter and changed the date to final